### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by kaedroho**

### CODE REVIEW

File: src/components/Comment/style.scss [hunk 1]
```diff
@@ -1,100 +1,204 @@
...
+        border-radius: 3px;
+        color: #fff;
+
+        &::placeholder {
+            font-style: italic;
+        }
+    }
+
+    &__reply-input {
+        margin-top: 20px !important;
```
Comment (jacobtoppm) [inline src/components/Comment/style.scss]: Just checking the !important isn't just for testing?

File: src/components/CommentReply/style.scss [hunk 2]
```diff
@@ -1,60 +1,119 @@
...
+            margin-right: 0px;
+        }
     }
 
-    &__text {
-        clear: both;
-        padding-left: 20px;
-        padding-bottom: 10px;
+    &__error {
+        background-color: #b4191f;
```
Comment (jacobtoppm) [inline src/components/CommentReply/style.scss]: Maybe worth replacing hardcoded colours for maintainability?

File: src/components/ModerationBar/index.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+import { Styling } from '../../utils/storybook';
+import APIClient from '../../api';
+import { setErrors, updateModerationState } from '../../actions/moderation';
+
+import ModerationBar from './index';
+import { ModerationErrorCode } from '../../state/moderation';
+
+export default { title: 'ModerationBar' };
+
+function RenderModerationBarForStorybok({ store }: { store: Store }) {
```
Comment (jacobtoppm) [inline src/components/ModerationBar/index.stories.tsx]: I do quite like the idea of using `Storybok`, but I think tbh we should maybe go with the more established `Storybook`.

**Review by emilytoppm — APPROVED** [review]
Generally looks great. One thing I think we should look at in the future is translation - if this is the package we're suggesting as the solution for comments in Wagtail workflow, we ought to look into supporting the same languages, I think.

File: src/components/ModerationBar/index.stories.tsx (hunk 3 above)
Comment (kaedroho) [reply src/components/ModerationBar/index.stories.tsx]: But `Storybok` is so much better! We save one byte every time we use it, which adds up you know...

File: src/components/CommentReply/style.scss (hunk 2 above)
Comment (kaedroho) [reply src/components/CommentReply/style.scss]: Done!

File: src/components/Comment/style.scss (hunk 1 above)
Comment (kaedroho) [reply src/components/Comment/style.scss]: It's overriding the rule set in `.comment textarea`

**Pull request merged**